### ISSUE TITLE
The chapter title always prefixed with its section number in the Table Of Contents

### ISSUE THREAD

**Issue by Zh40Le1ZOOB** [Michael-F-Bryan/mdbook-epub#84]
My ```book.toml``` included 
```toml
[output.epub]
no-section-label = true
```
But in the Table Of Contents, all chapters and sections was been numbered
![Screenshot_20231110_202224](https://github.com/Michael-F-Bryan/mdbook-epub/assets/97139537/16a2dc63-80a3-4bfd-8c65-6c93fc53a5f4)
![Screenshot_20231110_202358](https://github.com/Michael-F-Bryan/mdbook-epub/assets/97139537/819bdf57-d7f4-4b91-abfa-b57830984327)

**Comment by Zh40Le1ZOOB** [comment]
Not only that, toc-title isn't changes follw the languag option in ```boot.toml```

**Comment by blandger** [comment]
Could you pls give a link to book sources ?

**Comment by Zh40Le1ZOOB** [comment]
Here:
https://github.com/yndtk/YaoNiangDeTianKong-Fix

**Comment by dieterplex** [comment]
Looks like this feature is lost since merged. See
https://github.com/Michael-F-Bryan/mdbook-epub/commit/cdadaf6e#diff-01a1d809f0e7b93e591f14b055c3acc76e489ec56173eaead04003bb599bde3dL106

**Comment by Zh40Le1ZOOB** [comment]
Thank you very much! I'll open a new Pull Request to fix it.

**Comment by Zh40Le1ZOOB** [comment]
Thank you very much! I'll open a Pull Request to fix it.

**Comment by dieterplex** [comment]
Fixed by #85

`nav.xhtml` would like:
```
...
      <li><a href="introduction.html">简介</a></li>
      <li>
        <a href="1/0.html">第一卷 · 朦胧的世界</a>
        <ol>
          <li><a href="1/1.html">1 · 出走</a></li>
          <li><a href="1/2.html">2 · 小城市</a></li>
          <li><a href="1/3.html">3 · 独自一人，不知方向</a></li>
...
```

and `toc.ncx` would like:

```
...  <navMap>
    <navPoint id="navPoint-1">
      <navLabel>
       <text>简介</text>
      </navLabel>
      <content src="introduction.html"/>
    </navPoint>
    <navPoint id="navPoint-2">
      <navLabel>
       <text>第一卷 · 朦胧的世界</text>
      </navLabel>
      <content src="1/0.html"/>
      <navPoint id="navPoint-3">
        <navLabel>
         <text>1 · 出走</text>
        </navLabel>
        <content src="1/1.html"/>
      </navPoint>
...
```
